README.md

```diff
@@ -98,26 +98,18 @@ and GoTestWAF evaluating ModSecurity. There are two options to run the demo envi
 
 ### Running the demo using Docker
 
-1. Create the Docker network to link GoTestWAF and ModSecurity to. For example, to create the Docker network named `gotestwaf-modsecurity`:
+1. Clone this repository, go to the local folder, run the following commands to create a ModSecurity-based environment:
```

Comment:
Yes, it's possible to separate this step into two steps if it is required (or if it looks better), something like:
1. Clone the repository and change the directory to the cloned one
2. Run the following command to build modsec
```bash
docker pull owasp/modsecurity-crs:v3.3.2-nginx
docker run -p 8080:80 -p 8443:443 -d -e PARANOIA=1 -v ${PWD}/cmd/resources/default.conf:/etc/nginx/conf.d/default.conf --rm owasp/modsecurity-crs:v3.3.2-nginx
```

`make modsec` can be replaced with the command above (simply put, it's the same command from Makefile).

If you can do it, please, split this step as required.

Reply:
Can we use only the following four commands to run this demo with Docker?

```
docker pull owasp/modsecurity-crs:3.3.2-nginx
docker run -p <PORT_FOR_MODSECURITY>:80 -d -e PARANOIA=1 --rm owasp/modsecurity-crs:3.3.2-nginx
docker pull wallarm/gotestwaf
docker run -v ${PWD}/reports:/go/src/gotestwaf/reports --network="host" wallarm/gotestwaf --url=http://127.0.0.1:<PORT_FOR_MODSECURITY>
```

Since this demo is run using Docker, the step for cloning this repo looks extra. However, I see the default ModSec conf file and the one linked from our repository differ in the `listen`, `set`, `resolver`, `proxy_set`, `proxy_pass` directives. If this difference is critical for running ModSec-GoTestWAF demo, let's describe only one method to deploy ModSec: `make modsec`. If it is allowed to run ModSec with both default config and the one linked from our repo, let's explain the difference between these methods and provide commands for both methods.

Reply:
I believe this difference is quite critical.

In our case, to create a reproducible and stable environment we strongly recommend using the config file from the repository, otherwise, results can be unpredictable or something may go wrong (especially when OWASP can suddenly change default versions/tags/configurations/features included in their images).

So, as I think, `make modsec` can be considered as the proper way to run it.